vcd/resource_vcd_nsxt_standalone_vm_test.go

```diff
@@ -19,6 +19,15 @@ func init() {
 // TestAccVcdNsxtStandaloneVmTemplate tests NSX-T Routed network DHCP pools, static pools and manual IP assignment
 func TestAccVcdNsxtStandaloneVmTemplate(t *testing.T) {
 	preTestChecks(t)
+	if vcdShortTest {
+		t.Skip(acceptanceTestsSkipped)
+		return
+	}
+
+	vcdClient := createTemporaryVCDConnection()
+	if !vcdClient.Client.IsSysAdmin {
+		t.Skip(t.Name() + " only System Administrator can create Imported networks")
+	}
 
 	if testConfig.Nsxt.Vdc == "" || testConfig.Nsxt.EdgeGateway == "" {
 		t.Skip("Either NSXT VDC or edge gateway not defined")
```

Comment:
I think we should stick to "NSX-T" in messages. Personally, I'd also capitalize the Edge Gateway, just so it's poking in the eyes when you are scanning lines of text.

```suggestion
		t.Skip("Either NSX-T VDC or Edge Gateway not defined")
```

Reply:
Changed.